lib/java/com/google/android/material/tabs/TabLayout.java

```diff
@@ -2840,11 +2840,19 @@ private float approximateLineWidth(@NonNull Layout layout, int line, float textS
     SlidingTabIndicator(Context context) {
       super(context);
       setWillNotDraw(false);
-      selectedIndicatorPaint = new Paint();
       defaultSelectionIndicator = new GradientDrawable();
     }
 
     void setSelectedIndicatorColor(int color) {
+      if (color == 0) {
```

Comment:
a problem here is that the constant value for `Color.WHITE` is 0. so i think this change would disallow white for the selected indicator color?

Reply:
oh never mind, `Color.TRANSPARENT` is 0. i still wonder if that's an issue.

Reply:
thinking about this a little more 😅, i think this is actually ok. because if the color is `Color.TRANSPARENT`, then we set the paint to null, and we don't draw the indicator, which is sort of the same as drawing it transparently. will import this PR and change the `0` to `Color.TRANSPARENT` to make this a little more obvious.

Reply:
Awesome. Thank you. :+1: